test/ExpressiveInstallerTest/PromptForOptionalPackagesTest.php

```diff
@@ -90,8 +90,16 @@ public function testPromptForOptionalPackage(
 
     public static function assertPromptText($expected, $argument)
     {
-        $argument = is_array($argument) ? array_shift($argument) : $argument;
-        $message = sprintf('Expected prompt not received: "%s"', $expected);
-        self::assertThat(false !== strpos($argument, $expected), self::isTrue(), $message);
+        self::assertInternalType(
+            'string',
+            $argument,
+            'Questions must be a string since symfony/console:4.0'
+        );
+
+        self::assertThat(
```

Comment:
Why not `assertTrue`?

Reply:
Good question.

@mwop any reason you wrote it like this?

Reply:
I think you want to ask @weierophinney  ;-)

Reply:
Works for me :)

Reply:
Honestly, because it's how custom assertions are demonstrated in the manual: https://phpunit.de/manual/current/en/extending-phpunit.html

I'm fine with either approach.

Reply:
Hm... in the docs is just example, maybe a bit unfortunate, because `assertTrue` is one of the available assertions.